frontend/src/pages/(home)/checkout/page.tsx

```
@@ -293,12 +293,12 @@ function CheckoutPage() {
 
 									<div className="flex flex-1 justify-between gap-[10px]">
 										<div>
-											<h3 className="text-[15px] font-[500] leading-tight">{item.title}</h3>
+											<h3 className="text-[15px] leading-tight font-medium">{item.title}</h3>
```

ESLint

[error] Unsafe member access .title on an `any` value.

(ts-eslint/no-unsafe-member-access)